Library/Koala/Koala.swift

```diff
@@ -1813,30 +1818,29 @@ private func properties(reward reward: Reward, prefix: String = "backer_reward_"
   return result.prefixedKeys(prefix)
 }
 
-private func shareTypeProperty(shareType: String?) -> String {
+private func shareTypeProperty(_ shareType: UIActivityType?) -> String? {
   #if os(iOS)
-    guard let shareType = shareType else { return "" }
+    guard let shareType = shareType else { return nil }
 
-    switch shareType {
-    case UIActivityTypePostToFacebook:
+    if shareType == .postToFacebook {
       return "facebook"
-    case UIActivityTypeMessage:
+    } else if shareType == .message {
       return "message"
-    case UIActivityTypeMail:
+    } else if shareType == .mail {
       return "email"
-    case UIActivityTypeCopyToPasteboard:
+    } else if shareType == .copyToPasteboard {
       return "copy link"
-    case UIActivityTypePostToTwitter:
+    } else if shareType == .postToTwitter {
       return "twitter"
-    case "com.apple.mobilenotes.SharingExtension":
+    } else if shareType == UIActivityType("com.apple.mobilenotes.SharingExtension") {
       return "notes"
-    case SafariActivityType:
+    } else if shareType == SafariActivityType {
       return "safari"
-    default:
-      return shareType
+    } else {
+      return shareType.rawValue
     }
   #else
-    return ""
+    return nil
```

Comment:
i think that's the more correct expression of this function. tvos has no share sheet functionality and so it seems better to use `nil` to represent that